### PR TITLE
Added documentation for RemoteTensor and RemoteContext

### PR DESCRIPTION
**Pull request by ilyachur**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: docs/IE_PLUGIN_DG/Intro.md [hunk 1]
```diff
@@ -28,23 +28,27 @@ OpenVINO Plugin Library
...
-	which is backend-independent.
-	- Can import an already compiled graph structure from an input stream to an 
-	[compiled model](@ref openvino_docs_ov_plugin_dg_compiled_model) object.
+    - Provides information about devices of a specific type.
+    - Can create an [compiled model](@ref openvino_docs_ov_plugin_dg_compiled_model) instance which represents a Neural 
+    Network backend specific graph structure for a particular device in opposite to the ov::Model 
+    which is backend-independent.
+    - Can import an already compiled graph structure from an input stream to an 
+    [compiled model](@ref openvino_docs_ov_plugin_dg_compiled_model) object.
 2. [Compiled Modek class](@ref openvino_docs_ov_plugin_dg_compiled_model):
```
Comment (ilya-lavrenov) [inline docs/IE_PLUGIN_DG/Intro.md:36]:
```suggestion
2. [Compiled Model class](@ref openvino_docs_ov_plugin_dg_compiled_model):
```

File: docs/IE_PLUGIN_DG/RemoteContext.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+The implementation returns the remote context properties.
+
+@snippet src/remote_context.cpp remote_context:get_property
+
+
+### create_tensor()
+
+The method creates device specific remote tensor.
+
+@snippet src/remote_context.cpp remote_context:create_tensor
```
Comment (ilya-lavrenov) [inline docs/IE_PLUGIN_DG/RemoteContext.md:47]: Screenshot 2023-03-20 at 1 31 51 PM

File: docs/IE_PLUGIN_DG/RemoteContext.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+# Remote Context {#openvino_docs_ov_plugin_dg_remote_context}
```
Comment (ilya-lavrenov) [inline docs/IE_PLUGIN_DG/RemoteContext.md:1]: Screenshot 2023-03-20 at 1 29 53 PM

Looks like we need to add new pages to TOC on the left

File: docs/IE_PLUGIN_DG/RemoteTensor.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+The method returns tensor strides.
+
+### set_shape()
+
+The method allows to set new shapes for the remote tensor.
+
+### get_properties()
+
+The method returns tensor specific properties.
```
Comment (ilya-lavrenov) [inline docs/IE_PLUGIN_DG/RemoteTensor.md:83]: Screenshot 2023-03-20 at 1 33 20 PM

File: docs/IE_PLUGIN_DG/RemoteContext.md (hunk 2 above)
Comment (ilyachur) [reply docs/IE_PLUGIN_DG/RemoteContext.md:47]: Fixed

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**